src/runtime/Module.h

```diff
@@ -131,6 +131,29 @@ class ImportType {
         return static_cast<const MemoryType*>(m_type);
     }
 
+    std::string typeToString()
+    {
+        switch (m_importType) {
+        case Type::Function: {
+            return std::string("Function");
+        }
+        case Type::Global: {
+            return std::string("Function");
```

Comment:
Global is Function?

Reply:
I've pasted that wrongly and forgot to change it. It should be fixed now.